src/registries/auto_completes/pivot_auto_complete.ts

```diff
@@ -120,9 +120,13 @@ autoCompleteProviders.add("pivot_group_fields", {
     const rowFields = rows.map((groupBy) => groupBy.nameWithGranularity);
 
     const proposals: string[] = [];
-    const previousGroupBy = ["ARG_SEPARATOR", "SPACE"].includes(tokenAtCursor.type)
+    let previousGroupBy = ["ARG_SEPARATOR", "SPACE"].includes(tokenAtCursor.type)
       ? argGroupBys.at(-1)
       : argGroupBys.at(-2);
+    const isPositionalSupported = supportedPivotPositionalFormulaRegistry.get(pivot.type);
```

Comment:
So we have reference to positional pivots in o-srpeadsheet where there is absolutely no concept of positional pivot, I didn't know that, it sounds super good!🙃 Idk how we would do otherwise, but it kinda sucks 😢

Reply:
Also no tests then ? It sounds even worse 🙃

Reply:
It does suck indeed.
The test is in the enterprise PR, which I had forgotten to link in the task pad: https://github.com/odoo/enterprise/pull/71621